src/Arcus.Templates.WebApi/Serialization/HealthReportJson.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using GuardNet;
+
+// ReSharper disable once CheckNamespace
+namespace Microsoft.Extensions.Diagnostics.HealthChecks
+{
+    /// <summary>
+    /// Represents an alternative <see cref="HealthReport"/> model without the exception details so it can be exposed safely with OpenAPI.
+    /// </summary>
+    /// <remarks>
+    ///     This model should not be used within the domain and is just available for the serialization between trust boundaries.
+    ///     Use the <see cref="FromHealthReport"/> and <see cref="ToHealthReport"/> to switch between the two.
+    /// </remarks>
+    public class HealthReportJson
```

Comment:
Why do we have `Json`in the name ?  Suppose we use XML serialization, the model will be returned as Xml, so I would not include 'Json' in the name here.

Reply:
Ah, I see .... You want to avoid name-clashes.
Maybe `HealthStatusReport` ?

Reply:
Yeah, it's for the name clashes 😅 . Since our template is based on JSON serialization and we even have MVC extensions that only allows JSON serialization to the template, it seemed logically to also focus on JSON in the API model. If it were XML that needs to be serialized, then the startup functionality needs to be changed regardless and the `[JsonProperty]` stuff too. I kinda like it that it reflects in the name what it supposed to do, but we can change it indeed. `HealthStatusReport` will maybe confuse people, though, it would confuse me in any case 😅 .

Reply:
I just meant that imho it is not a good idea to include such an 'implementation detail' in the name of a type.

`HealthStatusReport`, `HealthResponse`, `HealthReportResponse`, `ApiHealthReport`, `HealthReportResult`, `HealthResult`, `HealthInformation` , .... are imho better then adding the Json suffix :)

Reply:
`ApiHealthReport` seems like a good one, then.